modules/date.js

```diff
@@ -0,0 +1,15 @@
+// Import necessary elements
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+// Function to display the current date and time
+export const dateTime = ()=> {
+  // Get the current time in Dublin timezone
+  const currentTime = DateTime.now().setZone('Europe/Dublin');
+
+  // Format the time
+  const formatedTime = currentTime.toFormat("MMMM d yyyy',' hh:mm:ss a");
+
+  // Display the time in the 'set-time' element
+  const elementTime = document.getElementById('set-time');
+  elementTime.innerText = formatedTime;
+}
```

Comment:
- [ ] [ OPTIONAL] Great stuff! :clap: :clap: How about implementing this in a `setInterval()` method so that you have your time updating every second? I think this will be more appealing to the users.